### DISCUSSION TITLE
Getting SWG running through AqualinkD 2.3.4

### DISCUSSION THREAD

**Discussion by jmp2000** [aqualinkd/AqualinkD#294] [Q&A] [answered]
Making good progress on the setup, running well and I have full control now.   One of the last problems I have encountered is that when I run the pool system from the Control Panel by the pumps, or through the OneTouch panel on the porch - the SWG runs (I get all the beeps, get salinity reading, get % output shown).   However, if I run the pool filter pump through AqualinkD it turns the pump on, but the SWG does not activate (no beeps, no salinity readings, no SWG% active - in the WebUI or on the system panels physically at the house.

My guess is something in the conf file is conflicting it and not sending the right signal to activate the SWG.  I've attached the Serial Protocol and my latest AqualinkD.conf file.   Would appreciate any advice!

![Screenshot 2024-05-18 130839](https://github.com/sfeakes/AqualinkD/assets/149847986/7166d242-75b8-49bb-a073-48533de265d6)
[aqualinkd 05-18-2024 - Copy.txt](https://github.com/sfeakes/AqualinkD/files/15366392/aqualinkd.05-18-2024.-.Copy.txt)

**Comment by jmp2000** [comment]
Version 2.3.6 got me going!   I think the keys are to use exactly the Serial IDs recommended in the Serial Logger.  That, combined with the 64-bit version for a Pi Zero 2 W has it going strong today.

**Comment by sfeakes** [comment, marked as answer]
Sorry for missing this before, and glad you got it working with the new release.
Also super happy someone (besides me) has tested the new 64 bit release binary.

**Reply by jmp2000** [reply]
Yes - I used the Raspberry Pi imager - using the Raspberry Pi OS Lite (64-bit) Version of Debian Bookworm (no desktop environment).  It says compatible with 3/4/400/5 (didn't explicitly say Pi Zero 2W in the description).  I'll let you know how it goes.
